scriptmodules/ports/wolf4sdl.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# This file is part of RetroPie.
+# 
+# (c) Copyright 2012-2015  Florian Müller (contact@petrockblock.com)
+# 
+# See the LICENSE.md file at the top-level directory of this distribution and 
+# at https://raw.githubusercontent.com/petrockblog/RetroPie-Setup/master/LICENSE.md.
+#
+
+rp_module_id="wolf4sdl"
+rp_module_desc="Wolf4SDL, port of Wolfenstein 3D"
+rp_module_menus="4+"
+rp_module_flags="dispmanx"
+
+function depends_wolf4sdl() {
+    getDepends libsdl1.2-dev
+}
+
+function sources_wolf4sdl() {
+    local cmd=(dialog --clear --backtitle "$__backtitle" --menu "Choose which version of Wolfenstein 3D game data you wish to use in Wolf4SDL." 22 86 16)
+    local options=(
+	1 "Shareware 1.4" 
+	2 "3D Realms (Apogee 1.4)" 
+	3 "Steam (GT/ID/Activision)"
+    )
+    local choice=$("${cmd[@]}" "${options[@]}" 2>&1 >/dev/tty)
+        if [[ -n "$choice" ]]; then
+            gitPullOrClone "$md_build" https://github.com/mozzwald/wolf4sdl.git
+            case $choice in
+                1)
+                    sed -i '/GOODTIMES/d' version.h
+                    ;;
+                2)
+                    sed -i '/GOODTIMES/d' version.h
+                    sed -i '/UPLOAD/d' version.h
+                    ;;
+                3)
+                    sed -i '/UPLOAD/d' version.h
+                    ;;
+            esac
+        else
+            break
+        fi
+}
+
+function build_wolf4sdl() {
+    make clean
+    make DATADIR="$romdir/ports/wolf3d/"
+    md_ret_require="$md_build"
+}
+
+function install_wolf4sdl() {
+     make install PREFIX="$md_inst"
+}
+
+function configure_wolf4sdl() {
+    mkRomDir "ports"
+    mkRomDir "ports/wolf3d"
+
+    # Create script to change file names to lower case
+    cat > "$romdir/ports/wolf3d/lower-script.sh" << _EOF_
+#!/bin/bash
+    for x in \`ls\`
+      do
+      if [ ! -f \$x ]; then
+        continue
+      fi
+      lc=\`echo \$x  | tr '[A-Z]' '[a-z]'\`
+      if [ \$lc != \$x ]; then
+        mv -i \$x \$lc
+      fi
+      done
+_EOF_
+
```

Comment:
What is this needed for - the prorgam needs lowercase game data or ? Also can just use "rename 'y/A-Z/a-z/' *"